prime-router/metadata/fhir_mapping/hl7/datatypes/CWE/Organization.yml

```diff
@@ -36,12 +36,23 @@ extension:
           expressionType: resource
           valueOf: datatypes/CWE/CodeableConcept
           specs: $BASE_VALUE
-    - expressionType: nested
-      condition: $cweOrganizationHL7Use NOT_NULL
+    - generateList: true
+      expressionType: nested
+      condition: $cweOrganizationHL7Field NOT_NULL
+      expressionsMap:
+        url:
+          type: SYSTEM_URL
+          value: hl7v2Field
+        valueString:
+          type: STRING
+          valueOf: $cweOrganizationHL7Field
+    - generateList: true
+      expressionType: nested
+      condition: $cweOrganizationHL7Component NOT_NULL
       expressionsMap:
         url:
           type: SYSTEM_URL
-          value: hl7-use
+          value: hl7v2Component
         valueString:
           type: STRING
-          valueOf: $cweOrganizationHL7Use
\ No newline at end of file
+          valueOf: $cweOrganizationHL7Component
```

Comment:
Should the variable name pattern stay the same across all the mappings (i.e `$hl7v2Component` vs `$cweOrganizationHL7Component`)? Seems like it would make it easier to document and understand for a FHIR sender

Reply:
So these would just be variable names and would not be surfaced to someone trying to compile a FHIR bundle.  This is not true across the board, but some of these cases were added because of nested schemas that were all setting the same variable